realtimenet/display.py

```diff
@@ -133,6 +134,47 @@ def display(self, img, display_data):
         return img
 
 
+class DisplayFPS(BaseDisplay):
+    """
+    Display camera fps and inference engine fps on debug window.
+    """
+
+    def __init__(self, neural_network, y_offset=20):
```

Comment:
Instead of passing the whole neural net, how about we add two arguments called `expected_camera_fps` and `expected_inference_fps`? I think that's everything we need. These arguments should be optional and default to `None`. If left to `None`, `self.running_delta_time_inference` and `self.running_delta_time_camera`  can be initialized to 0 and the text color will stay green.